### PR TITLE
[NCL-7659] Set the width of ContentBox responsively

### PR DESCRIPTION
**Pull request by DnsZhou**

### CODE REVIEW

**Comment by matedo1** [comment]
During the demo meeting we agreed on additional changes to be applied

**Review by matedo1 — CHANGES_REQUESTED** [review]
I think there are several other pages where `isResponsive` needs to be applied, right?

For example all Product related detail pages, Project detail page, Admin page, About page, etc. 

Please make sure to go through all pages and test them.

I would skip Variables page.

**Comment by DnsZhou** [comment]
Hi @matedo1 , following pages have also been integrated with the responsive feature:

AboutPage
AdministrationPage
ProductDetailPage
ProductMilestoneDetailPage (just for Attributes)
ProductVersionDetailPage (just for Attributes)
ProjectDetailPage

File: src/components/ContentBox/ContentBox.tsx [hunk 1]
```diff
@@ -18,6 +19,7 @@ interface IContentBoxProps {
   title?: string;
   contentHeight?: string;
   description?: IDescription;
+  isResponsive?: boolean;
```
Comment (matedo1) [inline src/components/ContentBox/ContentBox.tsx:22]: Add some TSDoc explaining when it should be used.

**Review by matedo1 — CHANGES_REQUESTED** [review]

File: src/components/ContentBox/ContentBox.tsx (hunk 1 above)
Comment (DnsZhou) [reply src/components/ContentBox/ContentBox.tsx:22]: Added accordingly

**Review by matedo1 — APPROVED** [review]

**Pull request merged**